pyperf/_runner.py

```diff
@@ -201,6 +201,9 @@ def __init__(self, values=None, warmups=None, processes=None,
                             help='Comma-separated list of environment '
                                  'variables inherited by worker child '
                                  'processes.')
+        parser.add_argument("--copy-env",
+                            dest="copy_env", action="store_true", default=False,
+                            help="Copy all env vars")
```

Comment:
```suggestion
                            help="Copy all environment variables")
```

You should also document the option at:
https://pyperf.readthedocs.io/en/latest/runner.html#misc